packages/admin-next/dashboard/src/v2-routes/shipping/service-zone-areas-edit/components/edit-region-areas-form/edit-service-zone-areas-form.tsx

```diff
@@ -0,0 +1,346 @@
+import { useForm } from "react-hook-form"
+import { zodResolver } from "@hookform/resolvers/zod"
+import {
+  ColumnDef,
+  createColumnHelper,
+  RowSelectionState,
+} from "@tanstack/react-table"
+import * as zod from "zod"
+
+import {
+  Alert,
+  Badge,
+  Button,
+  Checkbox,
+  Heading,
+  IconButton,
+  Text,
+  toast,
+} from "@medusajs/ui"
+import { RegionCountryDTO, RegionDTO, ServiceZoneDTO } from "@medusajs/types"
+import { useTranslation } from "react-i18next"
+import { XMarkMini } from "@medusajs/icons"
+
+import {
+  RouteFocusModal,
+  useRouteModal,
+} from "../../../../../components/route-modal"
+import { SplitView } from "../../../../../components/layout/split-view"
+import {
+  useCreateServiceZone,
+  useUpdateServiceZone,
+} from "../../../../../hooks/api/stock-locations"
+import { useEffect, useMemo, useState } from "react"
+import { useCountryTableQuery } from "../../../../regions/common/hooks/use-country-table-query"
+import { useCountries } from "../../../../regions/common/hooks/use-countries"
+import { countries as staticCountries } from "../../../../../lib/countries"
+import { useDataTable } from "../../../../../hooks/use-data-table"
+import { useCountryTableColumns } from "../../../../regions/common/hooks/use-country-table-columns"
+import { DataTable } from "../../../../../components/table/data-table"
+
+const PREFIX = "ac"
```

Comment:
question: What's this used for?

Reply:
Just to namespace query params for the table. It's not necessary in this case but it is a good practice if we have multiple tables on the page so we don't get params collision.

Reply:
Ahh got it